configurations/raft/enable_raft_experimental.yaml

```diff
@@ -1,3 +1,6 @@
 append_scylla_yaml: |
   experimental_features:
+    - alternator-streams
```

Comment:
remove those from here
this should only have raft (and even that might need to be changed)